src/flagged-disputes/voided-disputes.js

```diff
@@ -8,7 +8,7 @@ import { getNetworkType } from '../lib/web3-utils'
 import env from '../environment'
 
 const VOIDED_DISPUTES = {
-  rpc: new Map([[networks.rpc.court, new Map([])]]),
+  local: new Map([[networks.local.court, new Map([])]]),
```

Comment:
Could we also update the one at `precedence-campaign-disputes.js` that it was recently merged?